test/00-setup.js

```diff
@@ -19,6 +19,14 @@ var fixtures = path.resolve(__dirname, 'fixtures')
 var chars = ['a', 'b', 'c']
 var dirs = []
 
+for (var i = 0; i < 3; i ++) {
+  for (var j = 0; j < 3; j ++) {
+    for (var k = 0; k < 3; k ++) {
+      dirs.push('.git', chars[i] + '/' + chars[j] + '/' + chars[k])
```

Comment:
I think you'll also have to add some files (2 blocks down) or else these folders would be ignored by default anyway (since they're empty).